pages/api/handle-credentials.js

```diff
@@ -0,0 +1,32 @@
+export default async (req, res) => {
+  if (req.method !== 'POST') {
+    console.log('Only post request allowed');
+    res.status(400).json({});
+    return;
+  }
+
+  const body = req.body;
+
+  console.log('body::', body);
+
+  const result = await fetch(body.url, {
+    method: 'POST',
+    headers: {
+      accept: 'application/json',
+      'content-type': 'application/json',
+      Authorization: `Bearer ${process.env.DOCK_API_TOKEN}`,
```

Comment:
dont need both auth and dock-api-token here, just one

Reply:
```suggestion
    
```

Reply:
Fixed! Removed Authorization from request.